Formula/infer.rb

```diff
@@ -29,36 +35,69 @@ def install
     # Apple's libstdc++ is too old to build LLVM
     ENV.libcxx if ENV.compiler == :clang
 
-    opamroot = buildpath/"opamroot"
-    opamroot.mkpath
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["sqlite"].opt_lib/"pkgconfig"
+
+    # opamroot = HOMEBREW_CACHE/"opam"
+    opamroot = libexec/"opam"
```

Comment:
This variable only seems to be used once so we don't really need it and can set the value directly where its needed.